session.go

```diff
@@ -111,6 +135,54 @@ func (s *Session) Close() error {
 	return nil
 }
 
+// KillSession forces the session to stop by name.
+//
+// We should unsubscribe the session from all providers before
+// stopping the session (For proper session disabling see `Close` function).
+// We didn't find the way how to get a list of providers by session name
+// using windows API. We stop the session without unsubscribing it
+// from providers.
+func KillSession(name string) error {
+	const (
+		maxLengthSessionName = 1024
+		maxLengthLogfileName = 1024
+	)
```

Comment:
Why we need both if it was enough to allocate a space for name on connect?

And why we even need constants if we know length of the `name`?

Reply:
We don't know if this session was opened with the log file or not. (session could be opened without our library)
That's why we allocate memory for LogFile name too.

I changed `maxLengthSessionName` to `len(name) * 2` -_0_0_-

Reply:
Pls comment out the point about log file

__
> I changed maxLengthSessionName to len(name) * 2
```go
utf16Name, err := windows.UTF16FromString(s.config.Name)
...
l := len(utf16Name) * int(unsafe.Sizeof(utf16Name[0]))
```